apps/emqx/src/emqx_schema.erl

```diff
@@ -45,6 +46,7 @@
 -type timeout_duration_s() :: 0..?MAX_INT_TIMEOUT_S.
 -type timeout_duration_ms() :: 0..?MAX_INT_TIMEOUT_MS.
 -type bytesize() :: integer().
+-type mqtt_max_packet_size() :: 1..?MAX_INT_MQTT_PACKET_SIZE.
```

Comment:
nit: maybe rename `mqtt_max_packet_size` to `mqtt_packet_size`? 
(as it accepts value from 1 to max)

Reply:
Not sure if I agree with this one. A value of the type describes a max packet size.

Reply:
you were right! `mqtt_max_packet_size` describes better the purpose of it